.github/workflows/cd.yaml

```diff
@@ -47,9 +47,9 @@ jobs:
       with:
         fetch-depth: 0
     - name: Set up JDK 8
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v3.0.0
```

Comment:
```suggestion
      uses: actions/setup-java@v3
```

Reply:
I agree, low risk and less dependabot noise

Reply:
I was perplexed that dependabot did not offer the upgrade from `v2` to `v3`.  I specified precise version in hopes that dependabot would then recommend upgrades.  I'm OK with either.

Reply:
> in hopes that dependabot would then recommend upgrades

It will. The point is that it will then be offering PRs for `v3.0.1`, `v3.0.2`, etc., but the person receiving these PRs will just be blindly merging them anyway since the PRs will all be green (modulo flaky tests or CI outages), even if the update would in fact break the next release.

Reply:
> dependabot will not detect or report a new major version number

Oh wait I did not catch this before. Dependabot does not offer to update `v2` to `v3`? I know it will offer major version updates like https://github.com/jenkinsci/file-parameters-plugin/pull/85 in general. If true, this seems like a DB bug—is it filed?

Reply:
> Oh wait I did not catch this before. Dependabot does not offer to update `v2` to `v3`? I know it will offer major version updates like #85 in general. If true, this seems like a DB bug—is it filed?

Good suggestion that it should be filed as a bug report.  [actions/setup-java@v3.0.0](https://github.com/actions/setup-java/releases/tag/v3.0.0) was released Feb 24, yet none of the repositories that I was tracking have ever had a Dependabot pull request proposed to update from `actions/setup-java@v3``actions/setup-java@v2` to `actions/setup-java@v3`.  I think that means it is a dependabot bug

Reply:
We got a ton of PRs updating `actions/setup-java@v2` to `actions/setup-java@v3` this morning, as in https://github.com/jenkinsci/pipeline-input-step-plugin/pull/85, so I think the upstream bug has been fixed.